CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 This file is used to list changes made in each version of vcruntime.
 
+## 0.2.14:
```

Comment:
yes i could say so, v 1.0 seems appropriate. read note below for why the versions are aplenty in the change log.

FYI: in my environment at work i need to bump a version for every change i make so i stick in the minor versions until something is completely ready and then bump the major. anytime i push to git the version number has to be changed since it has a hook that automatically uploads to chef and to prevent untested code from executing we force version bumps and lock down prod version in environment so we only run whats tested.